cmds/boot/systemboot/main.go

```diff
@@ -33,6 +33,13 @@ var defaultBootsequence = [][]string{
 // Product list for running IPMI OEM commands
 var productList = [2]string{"Tioga Pass", "Mono Lake"}
 
+var selRecorded bool
+
+const (
+	// STD_TYPE     = 0x02
+	OEM_NTS_TYPE = 0xFB
```

Comment:
Would it make more sense to move these consts to the IPMI package?

Reply:
Move consts to the IPMI package.